### ISSUE TITLE
Use round numbered buttons at top to select pages of custom buttons

### ISSUE THREAD

**Issue by AquatikJustice** [dbkynd/controlcast#24]
Currently, we have a total of 80 customizable buttons (64 square + 16 rounded buttons).

What if we could use the top numbered buttons to select "pages" of buttons. That way we could organize things a little better and we'd get 576 assignable buttons (or you could make the rounded buttons on the right static between the pages and we'd have 512 customizable buttons).

Just a thought.

**Comment by dbkynd** [comment]
Oh, wow. You already used up all 80?
This would be quite the undertaking creating profiles mapped to buttons. And I don't think it would benefit the majority of users, most using ~20 keys or less.
The config is stored in the users appdata, so there is a different 'profile' for another logged in user.

**Comment by incredibleJo** [comment]
At first thank you pretty much for this tool! Thats exactly what i was looking for!
I would really appreciate a function to switch between pages too. There are so many functions in obs which are helpful to have on the launchpad like switching the scene, display videos or play sounds.
In fact it would be a cheaper and simple version of the outcoming elgato streamdeck in may and i think there enough users which would love it.
Maybe something for the future?
Thank you so far for the great work and please excuse my bad english. ;)

**Comment by AquatikJustice** [comment]
> Oh, wow. You already used up all 80?

Yes I have. I have 2 rows for just switching to different OBS scenes and that will be growing soon. Add to that sound fx and triggering different hotkey'd elements w/in those 16 scenes and suddenly 80 keys seems a bit...limiting.

**Comment by incredibleJo** [comment]
> This would be quite the undertaking creating profiles mapped to buttons.

I think it would be enough if there are more than one config file. For example the buttons on top or right side - as AquatikJustice has said.
Is it hard to realize that each button load its own config file?

**Comment by mr-nix** [comment]
I modified the get web request section to change between different config files.
I will upload a branch these days.

**Comment by mr-nix** [comment]
So finally here is my patcher: [Patcher Download](http://www.mediafire.com/file/t7x9jliy2x12ojx/controlcast_0.3.4_pages_patch_1.0.zip)

This patch removes the GET Request option and add multiple pages support!
After patching the current page is 8 to see the button configuration.
Page one is still your last button configuration. (save your configuration file anyway)

- currently supported number of pages: 8
- config naming convention: electron-config-1

Known bugs:
- creates icon every page switch

ToDo:
- More than 8 Pages
- Better Code ;P

**Comment by AquatikJustice** [comment]
@mr-nix Any chance you could upload your forked version to Github?

**Comment by mr-nix** [comment]
Sorry, I wasn't here for a long time. 
I just edited the installed software. Sorry but I don't have time to edit the source files. :/
